tests/core/test_expectation_suite.py

```diff
@@ -1216,19 +1210,19 @@ def profiler_config():
 def test_expectation_suite_copy(baseline_suite):
     suite_copy = copy(baseline_suite)
     assert suite_copy == baseline_suite
-    suite_copy.data_asset_type = "blarg!"
+    suite_copy.name = "blarg!"
```

Comment:
These tests need to change an arbitrary attribute. I'm using `name` instead of `data_asset_type` now.